docs/diagrams/DoneCryptoSequenceDiagram.puml

```diff
@@ -0,0 +1,116 @@
+@startuml
+'https://plantuml.com/class-diagram
+!include Styles.puml
```

Comment:
Can check the Styles.puml, you diagram should have white words in the boxes instead of blackwords, 
its hard to see
use this in styles
-> skinparam Participant {
    BorderThickness 0
    FontColor #FFFFFF
    FontName Arial
}

also hide the footbox which is the bottom box at the end of the lifeline